Configurations/WebKitProjectPaths.xcconfig

```diff
@@ -77,6 +77,9 @@ WK_PRIVATE_FRAMEWORK_STUBS_DIR_[sdk=iphone*] = $(PROJECT_DIR)/../../WebKitLibrar
 WK_PRIVATE_FRAMEWORK_STUBS_DIR_[sdk=appletv*] = $(PROJECT_DIR)/../../WebKitLibraries/WebKitPrivateFrameworkStubs/appletvos/$(WK_TARGET_TVOS_VERSION_MAJOR);
 WK_PRIVATE_FRAMEWORK_STUBS_DIR_[sdk=watch*] = $(PROJECT_DIR)/../../WebKitLibraries/WebKitPrivateFrameworkStubs/watchos/$(WK_TARGET_WATCHOS_VERSION_MAJOR);
 
+WK_DOWNLEVEL_FRAMEWORK_STUBS_DIR[sdk=macos*] = $(WK_DOWNLEVEL_FRAMEWORK_STUBS_DIR$(WK_MACOS_1400);
```

Comment:
It's called "downlevel stubs", but it's actually used for all pre-Sonoma builds, including open source ones. Let's just put these in the normal location, `WebKitLibraries/WebKitPrivateFrameworkStubs/Mac`.

Reply:
I don’t think that’s right. WebKitPrivateFrameworkStubs Is used for all open source builds, regardless of OS version. This makes sense, as the purpose is to allow open source builds to build with private frameworks.

I’m trying to allow both internal and open source builds to be able to build with a framework that isn’t part of the SDK, but will be present at runtime. This should happen on both internal and open source builds, but only on downlevel OS versions.

Further, I think it’s actually harmful to have internal builds link with WebKitPrivateFrameworkStubs, because for the frameworks in that directory, there should be a version in the internal SDK, and we want to make sure that if the internal framework changes symbols, that we get a build error rather than a runtime error.

So I really do think that WebKitPrivateFrameworkStubs should be a different thing than DownlevelFrameworkStubs.

Reply:
FWIW, we don't copy the `WebKitLibraries/` directory when running `installsrc` in production builds, so this kind of internal failure could only happen in debug/release Xcode builds.

Reply:
Oh, this means that this PR doesn't fix the problem, at least not entirely. Good point!

Reply:
So I need to move DownlevelFrameworkStubs into the Source/directory somewhere?

Reply:
It may be easiest to just add `ditto WebKitLibraries/DownlevelFrameworkStubs "$(SRCROOT)/WebKitLibraries/DownlevelFrameworkStubs"`

to the Makefile: https://github.com/WebKit/WebKit/blob/6a68b0c65d6c2504c8455ce9f88b870fb4d6ed94/Makefile#L44-L46.

Reply:
Either that, or change our `Makefile` to perform `installsrc` for more locations. Both changes would be easy to do; I didn't think through it enough to suggest which would be more elegant.

Reply:
(the latter is what Elliott showed how to do, I didn't see that comment when posting mine)